todo-app/src/index.js

```diff
@@ -1,5 +1,7 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./components/App";
```

Comment:
Normally it's customary to keep your React root (eg App.js) in the root directory, and put sub-components into a components directory. This way we know immediately who is the parent.